utils/crop_videos.py

```diff
@@ -26,7 +26,8 @@
 
 def extract_vid_uuid(vid_path: str) -> str:
     """Extracts uuid from video path"""
-    vid_name = vid_path.split("\\")[-1]
+    vid_path = vid_path.replace("\\","/")
```

Comment:
I think a better solution would be using os.path.split() here.
Could you test using that for better OS-agnostic compatability?

Reply:
The problem does not lie in the os, since python (on windows atleast) manages to work with forward and backward slashes. The problem comes from the format(/ OR \ separator) you give the --input and --output paths, since those don't get converted.

Reply:
Check out this article, just import pathlib and convert to path object [python pathing](https://medium.com/@ageitgey/python-3-quick-tip-the-easy-way-to-deal-with-file-paths-on-windows-mac-and-linux-11a072b58d5f)

Reply:
Ok did it, The input path gets now parsed using pathlib and the suffix is also removed using it